### PR TITLE
Featureform/gitbook fixes/shab

### PR DESCRIPTION
**Pull request by shabbyjoon**

### CODE REVIEW

File: docs/local_mode.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+When connecting or creating an Embeddinghub instance, a LocalConfig can be used instead.
 
 ```py
 import embeddinghub as eh
 
 hub = eh.connect(LocalConfig("directory"))
 ```
 
-This version of embedding hub works similarly to a traditional EmbeddingHub instance in that you can create and get spaces and write and read from them.
+This version of mbedding hub works similarly to a traditional EmbeddingHub instance in that you can create and get spaces and write and read from them.
```
Comment (simba-git) [inline docs/local_mode.md:13]: Missed one

File: docs/overview.md [hunk 2]
```diff
@@ -51,5 +51,5 @@ It works by transforming the user’s text and an image into an embedding in the
...
+The alpha version of the Embeddinghub can only be deployed in a single node configuration. It uses RocksDB to durably store embeddings and metadata, and uses HNSWLib to build approximate nearest neighbor indices. The python client also has the ability to use HNSWLib to build local embeddings but does not currently handle durable storage. The server communicates via gRPC, the proto service file can be found [here](https://github.com/featureform/embeddings/blob/main/embeddingstore/embedding_store.proto). All of the metadata is also stored in serialized protobuf form as defined [here](https://github.com/featureform/embeddings/blob/main/embeddingstore/embedding_store_meta.proto).
```
Comment (simba-git) [inline docs/overview.md:54]: Good catch

**Review by simba-git — CHANGES_REQUESTED** [review]
Missed one but looks great otherwise

File: docs/local_mode.md (hunk 1 above)
Comment (shabbyjoon) [reply docs/local_mode.md:13]: my bad, thanks!

**Pull request merged**